sdks/python/apache_beam/runners/common.py

```diff
@@ -332,31 +437,45 @@ def __init__(self,
       kwargs: keyword side input arguments (static and placeholder), if any
```

Comment:
<!--thread_id:cc_151534742_t; commit:7ca389e990c4482eec47aecdc1913eeee0b169f9; resolved:1-->
<!--section:context-quote-->
> **chamikaramj** wrote:
> We need to pass in a customer OutputProcessor when invoking SDF.process() instead of using the default output processor since output has to be handled at ProcessFn.

<!--section:body-->
Done.